apps/filecopy/src/main/java/com/datatorrent/apps/copy/HDFSFileCopyApp.java

```diff
@@ -19,14 +19,14 @@
 
 package com.datatorrent.apps.copy;
```

Comment:
Should this be relocated to org.apache.malhar.apps.copy (possibly as part of a follow-up PR). If yes, can you file JIRA?